haystack_experimental/tools/__init__.py

```diff
@@ -0,0 +1,7 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from .component_schema import create_tool_parameters_schema
+
+__all__ = ["create_tool_parameters_schema"]
```

Comment:
- I would not export this function here if possible - see https://github.com/deepset-ai/haystack/issues/8650
- I would prefer to make this method internal and also all others in component_schema.py. They should not be user-facing and if we make them internal, we are then free to change them at any time if needed.

Reply:
Ok makes sense, will do 🙏